### PR TITLE
Update Readme for multi gpu train

### PR DESCRIPTION
**Pull request by diazandr3s**
Signed-off-by: Andres <diazandr3s@gmail.com>

Fixes  https://github.com/Project-MONAI/tutorials/issues/885.

### Description
Update Readme for multi GPU running

### CODE REVIEW

File: deepedit/ignite/README.md [hunk 1]
```diff
@@ -70,6 +70,14 @@ python ./train.py
     --output      deepedit_model/ \
     --epochs      100
 
+# Train a DeepEdit model using multi gpu
+python -m torch.distributed.launch \
```
Comment (Nic-Ma) [inline deepedit/ignite/README.md:74]: Maybe better to use the new PyTorch API?
https://github.com/pytorch/pytorch/blob/master/torch/distributed/run.py

Thanks.

Comment (diazandr3s) [reply deepedit/ignite/README.md:84]: Agree

File: deepedit/ignite/README.md [hunk 2]
```diff
@@ -70,6 +70,24 @@ python ./train.py
...
 
+# Train a DeepEdit model using multi gpu
+
+# Using new PyTorch API
+torchrun --standalone \
+       --nnodes=1 \
+       --nproc_per_node=`nvidia-smi -L | wc -l` \
+       --node_rank=0 \
+       --master_addr="localhost" \
+       --master_port=1234 train.py --input /PATH_TO_DATASET/ --output deepedit_model/ --epochs 100 --multi_gpu True
```
Comment (Nic-Ma) [inline deepedit/ignite/README.md:81]: I think no need the `master_addr`, `master_port`, `node_rank` args for the `standalone` running?
Refer to: https://github.com/pytorch/pytorch/blob/master/torch/distributed/run.py#L80

Thanks.

Comment (diazandr3s) [reply deepedit/ignite/README.md]: agree

**Review by Nic-Ma — APPROVED** [review]

**Pull request merged**